frontend/src/components/Inquiries/InquiriesTable.tsx

```diff
@@ -6,32 +6,46 @@ import { DataTable } from "../Common/Table.tsx"
 import { columns } from "./InquiriesTable.columns.tsx"
 
 const InquiriesTable = () => {
-  const { data: inquiries, isLoading } = useInquiries()
+  const { data: inquiries, isLoading, refetch } = useInquiries()
 
   // Sort inquiries from Newest to oldest
   const sortedInquiries = useMemo(() => {
     if (!inquiries?.data) return []
     return inquiries.data.sort((a: InquiryPublic, b: InquiryPublic) => {
-      return new Date(b.created_at).getTime() - new Date(a.created_at).getTime()
+      return (
+        (a.scheduled_inquiry ? a.scheduled_inquiry.rank : -1) -
+        (b.scheduled_inquiry ? b.scheduled_inquiry.rank : -1)
+      )
     })
   }, [inquiries])
 
   const handleRowClick = (inquiry: InquiryPublic) => {
     console.log("Row clicked:", inquiry)
   }
 
+  const refetchInquiries = async () => {
+    await refetch()
+    // const target = inquiries?.data?.find(i => i.id == inquiry.id)
```

Comment:
what's the deal here?

Reply:
I did this in the spirit of MVP for now. I am just refreshing the whole list on every update since the list is small currently. This will be updated in my next PR to only refresh the updated item in the list which will require moving to decimal ranks.